zerver/webhooks/freshstatus/doc.md

```diff
@@ -0,0 +1,79 @@
+Receive Freshstatus notifications in Zulip!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to your Freshstatus dashboard. Click **Settings** and then
+   **Integrations**. Go to **Webhooks** and click **Manage** and
+   then click **New Webhook**.
+
+1. Under **Webhook Name** enter `Zulip` or anything else you wish.
+   Enter whatever you like under **Description**.
+
+1. Under **Condition** you can set which events and components you would
+   like to alert for. Under **Action** ensure that **Request Type** is
+   set to `POST`. Under **Callback URL** enter the **URL** you
+   generated earlier.
+
+1. Leave **Requires Authentication** unticked and set **Content** to
+   **Advanced**.
+
+1. Under **Incident JSON** ensure the following is set:
+
+      ```
+      {
+         "id": "{id}",
+         "title": "{title}",
+         "description": "{description}",
+         "start_time": "{start_time}",
+         "end_time": "{end_time}",
+         "is_private": "{is_private}",
+         "source": "{source}",
+         "affected_services": "{affected_services}",
+         "notification_options": "{notification_options}"
+      }
+      ```
+
+1. Under **Maintenance JSON** ensure the following is set:
+
+      ```
+      {
+         "id": "{id}",
+         "title": "{title}",
+         "description": "{description}",
+         "start_time": "{start_time}",
+         "end_time": "{end_time}",
+         "is_private": "{is_private}",
+         "source": "{source}",
+         "affected_services": "{affected_services}",
+         "notification_options": "{notification_options}",
+         "scheduled_start_time": "{scheduled_start_time}",
+         "scheduled_end_time": "{scheduled_end_time}",
+         "is_auto_update_status_on_scheduled_start": "{is_auto_update_status_on_scheduled_start}",
+         "is_auto_update_status_on_scheduled_end": "{is_auto_update_status_on_scheduled_end}",
+         "is_auto_update_component_status_on_scheduled_start": "{is_auto_update_component_status_on_scheduled_start}",
+         "is_auto_update_component_status_on_scheduled_end": "{is_auto_update_component_status_on_scheduled_end}"
+      }
+      ```
+
+1. Under **Incident/Maintenance Note JSON** ensure the following is set:
+
+      ```
+      {
+         "id": "{note_id}",
+         "title": "{title}",
+         "incident_id": "{note_incident_id}",
+         "incident_status": "{note_incident_status}",
+         "message": "{note_message}",
+         "status": "{note_status}",
+         "is_private": "{note_is_private}",
+         "notification_options": "{note_notification_options}"
+      }
+      ```
+
+1. Click **Save**. You can then click **Test** to check it is configured correctly.
```

Comment:
Finally, click **Save**. Optionally, you may click **Test** to check if the webhook was configured correctly.